src/plotting/plotSettings/moreOptionsToggle.component.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Box, IconButton } from '@mui/material';
+import { MoreVert } from '@mui/icons-material';
+import MoreOptions from './moreOptions.component';
+import type { MoreOptionsProps } from './moreOptions.component';
+
+// from react-colorful PopoverPicker demo :https://codesandbox.io/s/opmco?file=/src/PopoverPicker.js
+
+// Improved version of https://usehooks.com/useOnClickOutside/
+const useClickOutside = (
```

Comment:
Isn't this the same code as in `colourPicker`? If it is, we should extract it to its own file to reduce code repetition